src/todo/CreateTodo.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function InputAndBtn({setInitialText, clicked}){
+    
+    return(
+        <div>
+            <input type="text" id="fname" name="fname" placeholder="Input" onChange={(event)=>setInitialText(event.target.value)}></input>
+            <button type="button" onClick={clicked}>Click</button>
+        </div>
+    )
+}
+
+export default InputAndBtn;
```

Comment:
Usually the default exported variables/functions should use the same name as the file. In this case, the function should be named `CreateTodo`